### PR TITLE
Update view and scene in api request for receiving items

### PR DESCRIPTION
**Pull request by chiaberry**
Knack added some new security measures

> We've taken a look and it appears that the form you're using for this call (view_698) is a form that Adds a new record, and doesn't update one. As such, since it's an Add form, you would need to use a POST call, and using PUT calls will no longer work.
>To give you some more context on this, it was previously possible to use Add forms with a PUT call, however, this shouldn't have been possible. It has been identified as a security gap by our team and was addressed by the update.

So we need to update the api url for updating received items. The code in production did not match the code we have here in github, so I am updating it to match. 

The only changes to the code are on lines 368, 369, 733 and 735.

### CODE REVIEW

File: code/finance-puchasing/finance-purchasing.js [hunk 1]
```diff
@@ -379,6 +365,9 @@ var invoicesView = "view_282";
 // Invoice Items API view form scene and view
 var invoicesAPIViewConfig = { scene: "scene_123", view: "view_698" };
 
+// Invoice Items API view form scene and view
+var itemsAPIViewConfig = { scene: "scene_336", view: "view_802" };
```
Comment (chiaberry) [inline code/finance-puchasing/finance-purchasing.js:369]: here is the new form config

File: code/finance-puchasing/finance-purchasing.js [hunk 2]
```diff
@@ -741,9 +730,9 @@ function handleCreateInvoiceClick(event, id, view) {
       type: "PUT",
       url:
         "https://api.knack.com/v1/scenes/" +
-        invoicesAPIViewConfig.scene +
+        itemsAPIViewConfig.scene +
         "/views/" +
-        invoicesAPIViewConfig.view +
+        itemsViewConfig.view +
```
Comment (chiaberry) [inline code/finance-puchasing/finance-purchasing.js:735]: updating to use the new scene and view

File: code/finance-puchasing/finance-purchasing.js [hunk 3]
```diff
@@ -858,3 +847,11 @@ if (!isUserRoleOnlyUser()) {
...
   );
 }
+
+/**** Remove extra whitespace from stock number input ****/
+$(document).on("knack-view-render.view_794", function (event, page) {
+  $("#field_720").on("change", function () {
+    var val = $("#field_720").val();
+    $("#field_720").val(val.trim());
+  })
+});
```
Comment (johnclary) [inline code/finance-puchasing/finance-purchasing.js:857]: dis was me. i forgot to update github 🙈

**Review by johnclary — APPROVED** [review]
🚢!!!!!

**Review by dianamartin — APPROVED** [review]

**Review by dianamartin — APPROVED** [review]

**Pull request merged**